src/test/java/study/SetTest.java

```diff
@@ -0,0 +1,51 @@
+package study;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@DisplayName("Set 테스트")
+public class SetTest {
+
+	private Set<Integer> numbers;
+
+	@BeforeEach
+	void setUp() {
+		numbers = new HashSet<>();
+		numbers.add(1);
+		numbers.add(1);
+		numbers.add(2);
+		numbers.add(3);
+	}
+
+	@Test
+	@DisplayName("set 크기 테스트")
+	void sizeCheck() {
+		int size = this.numbers.size();
+		assertThat(size).isEqualTo(3);
+	}
```

Comment:
hasSize() 를 사용해볼 수도 있을 것 같아요! 😄 
```suggestion
	@Test
	@DisplayName("set 크기 테스트")
	void sizeCheck() {
		assertThat(this.numbers).hasSize(3);
	}
```

Reply:
assertThat에 주입한 인자 타입에 따라 제공되는 메서드를 적극 활용하도록 해보겠습니다. 리뷰 감사드립니다^^